gtsam/hybrid/tests/testHybridBayesNet.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include "Switching.h"
 #include "TinyHybridExample.h"
+#include "gtsam/nonlinear/NonlinearFactor.h"
```

Comment:
This shouldn't be necessary since we've included it via Switching.h.

Reply:
https://google.github.io/styleguide/cppguide.html#Include_What_You_Use